.circleci/config.yml

```diff
@@ -419,52 +390,37 @@ jobs:
               php run-tests.php -p $(which php) --show-all -d zend_extension=xdebug-<< parameters.xdebug_version_two >>.so ../../tests/xdebug/<< parameters.xdebug_version_two >>
             fi;
 
-  unit_tests:
+  test: &TEST_BASE
     parameters:
-      docker_image:
-        type: string
-      valgrind_config:
+      php_major_minor:
+        # Expected in the format: <major>.<minor>, e.g. 5.6
         type: string
-        default: musl
-      lib_curl_command:
+      make_target:
         type: string
-        default: none
+      # Can we remove it?
```

Comment:
will push this comment removal when removing the `_tmp` removal.

Reply:
addressed here https://github.com/DataDog/dd-trace-php/commit/ea0ff4a757f8227cdfb89a754668fda0392ca9e1